react/AppLinker/index.jsx

```diff
@@ -0,0 +1,127 @@
+/* global __TARGET__ */
+
+import React from 'react'
+import {
+  checkApp,
+  startApp,
+  isMobileApp,
+  isMobile,
+  openDeeplinkOrRedirect
+} from 'cozy-device-helper'
+
+import expiringMemoize from './expiringMemoize'
+import { appShape } from 'cozy-ui/transpiled/react/propTypes'
+import { NATIVE_APP_INFOS } from 'cozy-ui/transpiled/react/AppLinker/native'
```

Comment:
I don't know if cozy-ui has to know about that. But I don't know how we can have these informations dynamically. Any idea?

Reply:
I think it is OK that cozy-ui has this knowledge, it does not change much.